src/schedule.cpp

```diff
@@ -304,6 +304,52 @@ struct stream_info
         return result;
     }
 
+    void
+    compute_conflict_table(instruction_ref ins,
+                           const std::vector<std::vector<instruction_ref>>& merge_ins,
+                           std::unordered_map<instruction_ref, std::unordered_set<instruction_ref>>&
+                               thrd_conflict_table)
+    {
+        int size = static_cast<int>(merge_ins.size());
+        std::unordered_set<instruction_ref> checked_ins1_set;
```

Comment:
Why is this called `checked_ins1_set`? There doesn't seem to be a `checked_ins2_set`?

Reply:
instructions in the concurrent table meets that requirement